File: Kendo_React/src/types/savedView.ts
export interface SavedView {
    id: string;
    name: string;
    departmentId: number;
    employmentTypeId: number;
    locationId: number;
    isDefault: boolean;
    createdAt: string;
    updatedAt: string;
}

export interface SavedViewInput {
    name: string;
    departmentId: number;
    employmentTypeId: number;
    locationId: number;
    isDefault: boolean;
}

File: Backend/Models/SavedView.cs
namespace Backend.Models;

public class SavedView
{
    public Guid Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public int? DepartmentId { get; set; }
    public int? EmploymentTypeId { get; set; }
    public int? LocationId { get; set; }
    public bool IsDefault { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}
